include/cute/atom/copy_traits_xe.hpp

```diff
@@ -536,7 +629,25 @@ struct Copy_Traits_<XE_2D_U8x16x32_LD_N, args_t...>
 
   template <class... ArgT>
   Copy_Traits_(ArgT... args)
-      : XE_2D_LD_Unpack<XE_2D_U8x16x32_LD_N, args_t...>(args...) {}
+      : XE_2D_LD_Unpack<XE_2D_Packed_U8x16x32_LD_N, args_t...>(args...) {}
+};
+
+template <class... args_t>
+struct Copy_Traits_<XE_2D_Packed_U8x32x32_LD_N, args_t...>
+    : XE_2D_LD_Unpack<XE_2D_Packed_U8x32x32_LD_N, args_t...> {
+  using ThrID = Layout<_16>;
+  // Map from (src-thr,src-val) to bit
+  using SrcLayout = Layout<Shape <_16,Shape <_8,  _2, _32>>,
+                           Stride< _0,Stride< _1,_128,_256>>>;
+  // Map from (dst-thr,dst-val) to bit
+  using DstLayout = Layout<Shape <_16,Shape <_8,  _2, _32>>,
+                           Stride<_16,Stride< _1,_128,_256>>>;
```

Comment:
It looks like `XE_2D_Packed_U8x32x32_LD_N` and `XE_2D_U8x32x32_LD_N` have the same `*Layout` traits. Is that expected?

Reply:
I'll check out the `copy_debug` tool to verify why they look similar (they were same when you commented) & will report back with any findings. Thanks!